metadata_mapper/mappers/mapper.py

```diff
@@ -131,6 +130,19 @@ def collate_fields(self, fieldlist):
 
         return collated
 
+    def first_string_in_field(self, field):
+        """
+        TODO: get approval on method name
+
+        Fetches a field value from source_metadata, and returns it if it's a string, or the first
+        item if it's a list
+        """
```

Comment:
This needs an approved name, @amywieliczka

Reply:
!!! Approved 💯